docs/docs/Support/security.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Security notice
+slug: /security
+---
+
+The Langflow UI is an IDE and code execution platform, which means that Langflow is inherently capable of executing arbitrary, developer-provided code.
+By design, the Langflow UI includes a code editor allowing developers to author and execute arbitrary Python with full access to the host Langflow backend process, filesystem, and network.
+In addition, various components wrap and make use of code execution, in some cases using AI models to generate the code to be executed based on user input.
+
+Given these product features, the Langflow application makes no attempt to prevent malicious activity between different users of a given Langflow process, or to secure the execution environment of the Langflow process, such as preventing access to portions of the local disk. To the extent that limitations are imposed on the access provided to different users, such as limiting flow visibility to a user's flows, these limitations are intended to improve usability, not to impose secure access policies.
+For example, users of Langflow can always access the underlying database connection directly.
+
+The security of code and flows written using Langflow is the developer’s responsibility.
+Langflow does not and can not prevent developers from building insecure applications.
+In the same way that Python can’t prevent developers from writing services with SQL-injection vulnerabilities, Langflow can’t prevent developers from writing flows that leak user’s PII, fail to sanitize input, etc.
+Langflow makes efforts to ensure that components and flows are secure by default and behave as expected, but application behavior is ultimately the developer’s responsibility. 
+
+Langflow’s cloud security flows from infrastructure.
+Cross-tenant isolation is provided by isolating instances of Langflow to individual tenants, not by the Langflow application itself.
+Within a Langflow cloud container, users are given full access to the process, disk, and network.
+Security restrictions are enforced between cloud containers, and between containers and the host instance and network.
```

Comment:
```
Security restrictions are enforced between cloud containers, and between containers and the host instance and network.
```

Does _LF_ enforce this or is this enforced by container orchestration/cluster management?